features/FxA-53-send-sms/README.md

```diff
@@ -0,0 +1,444 @@
+
+## Hypothesis
+
+If we offer to download via SMS to remove friction around setting up a second device for firstrun users on desktop, then we will see an increase in multi-device users within the first 7 days of registration because we have observed that less than [30% of registration](https://next.adjust.com/#/statistics/overview/default/gqrwnbypw28p/hqnqs2?from=2016-10-01&to=2016-10-31&range=last-month&sort=installs&reverse=true) (65k clicks / 210k registration / month) engage with our mobile download buttons and [at best 26%](https://sql.telemetry.mozilla.org/queries/1254#2173) (8% 7d md / 30% ctr) of them follow through to an install within 7 days after clicking.
+
+## Metrics
+
+We will know our hypothesis to be true when we see the proportion of multi-device users increase after 48h and 7d of registration to FxA+Sync on firstrun.
+
+<img src="./image_0.png" / alt="New signups that connect another device" />
+
+From: https://sql.telemetry.mozilla.org/queries/1254#2173
+
+We will also want to keep an eye out for:
+
+* **Email confirmation rate**: we should not impact this negatively in any way.
+
+    * Measured with telemetry and visualized in re:dash
+
+* **Volume of SMS sent**: Will confirm if users are adopting new mechanism.
+
+    * Measured with SMS provider
+
+* **SMS send rate**: Will confirm if users are adopting new mechanism.
+
+* **Click rate**: Will allow to see if messages are delivered and users are clicking on them.
+
+* **Install rate from SMS**: Will allow to see if users are successfully installing after install click on link in SMS
+
+* **Install rate** from confirmation page on mobile: If users confirmed their email on mobile, we will drive users to install via smart banner. (install rate)
+
+### Flow events
+
+<table>
+  <tr>
+    <td>flow.sms.view</td>
+    <td># of people that see the send_sms screen</td>
+  </tr>
+  <tr>
+    <td>flow.sms.engage</td>
+    <td># of people that interact with form.</td>
+  </tr>
+  <tr>
+    <td>flow.sms.submit</td>
+    <td># of people that try to send a message.</td>
+  </tr>
+  <tr>
+    <td>sms.1.sent</td>
+    <td># of people that we successfully send a message to.</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.learn_more</td>
+    <td># of people that click on "learn more"</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.why</td>
+    <td># of people that click on “why is this required”</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.maybe_later</td>
+    <td># of people that click “maybe later”</td>
+  </tr>
+</table>
+
+
+### Formula
+
+<table>
+  <tr>
+    <td>SMS form submit rate</td>
+    <td>flow.sms.submit / flow.sms.view</td>
+  </tr>
+  <tr>
+    <td>SMS form submit success rate</td>
+    <td>sms.1.sent / flow.sms.submit </td>
+  </tr>
+  <tr>
+    <td>% of people that sign in that have gone through this flow</td>
```

Comment:
Do we know yet how this will be measured? I think it will be important to determine the priority of phase 3 and 4. But perhaps the mechanism to track this is part of phase 3 where we intend to pass parameters to the app. Let me know your thoughts.